src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/SessionTemplateRepository.kt

```diff
@@ -83,6 +79,18 @@ interface SessionTemplateRepository : JpaRepository<SessionTemplate, Long> {
     @Param("visitsToDate") visitsToDate: LocalDate?,
   ): List<VisitCountsByDate>
 
+  @Query(
+    "select count(*) from visit v " +
+      " WHERE v.session_template_reference = :reference" +
+      "   AND v.visit_start >= :sessionDate AND v.visit_start <= :sessionDate" +
+      "   AND visit_status IN ('CANCELLED') AND v.outcome_status != 'SUPERSEDED_CANCELLATION'",
```

Comment:
I think we need to add v.outcome_status IS NULL to v.outcome_status != 'SUPERSEDED_CANCELLATION' like the query above (line 56)

Reply:
Already done,... just did not push up the correct code,...